data/scenarios/Testing/475-wait-one.yaml

```diff
@@ -30,7 +31,7 @@ robots:
       - logger
       - clock
     program: |
-      log "I shall sleep"; wait 1; log "I have awoken"
+      log "I shall sleep"; wait 2; log "I have awoken"
```

Comment:
Does this test still make sense, now that we wait for two ticks and  newly in solution for one? 🤔